app/views/posts/index.html.erb

```diff
@@ -1,2 +1,19 @@
-<h1>I am users posts</h1>
-<p>Users Post</p>
\ No newline at end of file
+ <div class="container mt-5">
+ <% @posts.each do |post| %>
+  <div class="row justify-content-between m-auto border border-dark mb-1">
+  <%= link_to user_post_path(@user,post.id), class:"text-decoration-none text-dark"  do%>
+    <h2><%= post.title  %></h2>
+  <%end%>
+    <p><%= truncate(post.text, length: 20) %></p>
+    <span class="d-flex justify-content-end">Comments: <%= post.commentscounter %>,Likes: <%= post.likescounter %></span>
+  </div>
+  <% post.last_five_comments.each do |comment| %>
+   <div class="row justify-content-between m-auto border border-dark mb-3">
+   <p><%= comment.author.name %>: <%= comment.text %></p>
+   </div>
+   <% end %>
+  <% end %>
+  <div class="row col-2 justify-content-between m-auto mb-1">
+    <button class="btn btn-primary" type="button">Pagination</button>
+  </div>
+   </div>
```

Comment:
![image](https://github.com/sajibarafatsiddiqui/BlogMaster/assets/93039359/5143a3c2-9906-4dcd-aeb3-ee7dcbce5cd9)

- [ ] It would be nice if you make the comments and likes counter to work to make your app work as expected :+1: right now you can see that there are two comments but the comments counter is not counting them :fearful:

Reply:
corrected